config/jobs/kubernetes/node-problem-detector/node-problem-detector-presubmits.yaml

```diff
@@ -187,7 +187,7 @@ presubmits:
         - name: ZONE
           value: us-central1-a
         - name: IMAGE_FAMILY
-          value: cos-stable
+          value: cos-97-lts
```

Comment:
please add comment here with the bug link

Reply:
Done.